docs/docs/understand/rule_evaluation.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Rule evaluations
+sidebar_position: 50
+---
+
+# Rule evaluations
+
+When Minder evaluates the [rules in your profiles](profiles.md), it records the state of those evaluations. When those rules are not satisfied because the criteria that you defined was not met, it will issue [alerts](alerts.md).
+
+Minder evaluates the rules in your profile:
+* When the repository is registered
+* When the profile is updated
+* When activity occurs within the repository
+
+In a rule evaluation, you'll see:
+* The time that a rule evaluation was performed
+* The entity that was examined (a repository, artifact, or pull request)
+* The rule that was evaluated
+* The status of the evaluation
+* Whether an [alert](alerts.md) was opened
+* Whether [automatic remediation](remediations.md) was performed, and if so, its status
+
+### Viewing rule evaluations
+
+To view the rule evaluations, run [`minder history list`](../ref/cli/minder_history_list.md). You can query the history to only look at certain entities, profiles, or statuses.
+
+### Evaluation status
+
+The _status_ of a rule evaluation describes the outcome of executing the rule against an entity. Possible statuses are:
+
+* **Success**: the entity was evaluated and is in compliance with the rule. For example, given the [`secret_scanning`](../ref/rules/secret_scanning) rule, this means that secret scanning is enabled on the repository being evaluated.
```

Comment:
There is also "Pending" which means "this rule was not evaluated yet". I'm not sure if it's even worth documenting because the for repos the only way to have a pending result is to create a profile before you register a repo.

Reply:
I think that it's worth being complete here; I added a bullet point, LMK what you think.